modules/virt-customizing-storage-profile.adoc

```diff
@@ -6,7 +6,10 @@
 
 = Customizing the storage profile
 
-You can customize any storage profile. Use the `spec` section of the storage profile to provide specific attributes when creating an underlying persistent volume claim (PVC) for a data volume.
+You can customize any storage profile to optimize performance and automate setting of specific attributes. Use the `spec` section of the storage profile to provide specific attributes when creating an underlying persistent volume claim (PVC) for a data volume.
+
+.Prerequisites
+* Each storage provisioner supports specific file structures and YAML attributes. Specifying an incompatible customization causes provisioning of the volume to fail and leaves the PVC in *Pending* state. For example, with some provisioners, `volumeMode` cannot be `Filesystem`.
 
```

Comment:
Leaving out the "the" before CDI is new to me. Is there a guideline on this. In other places it is used, which to me makes sense as it is "the container data interface".  But if this is the standard, that's cool. I changed it in this case.

On the callouts in the second YAML, yes, Adam and I thought they provided some value.

Thanks Pan, this was very helpful.

Reply:
Right, when we write out "Containerized Data Importer" we usually put a "the" in front of it, but when we use the CDI acronym, @aglitke requested that we just call it "CDI" instead of "the CDI." I'm not sure I'll be able to find the PR where this was discussed but maybe Adam can confirm?

Reply:
Gotcha, no worries. Thanks for checking it.